sms-checker/notification_lambda.py

```diff
@@ -0,0 +1,14 @@
+'''This script contains the lambda handler for this pipeline'''
+import logging
+from checker import get_subscribers
+from message import send_email,send_sms
+
+def lambda_handler(event,context) -> None:
```

Comment:
Maybe I'm missing something here, but will the lambda_handler function run the pipeline? It seems to only be defining a logging object and its config.